wire-ios-data-model/Source/Model/Conversation/ZMConversation+Internal.h

```diff
@@ -116,7 +116,6 @@ NS_ASSUME_NONNULL_END
 @property (nonatomic, nullable) NSUUID *teamRemoteIdentifier;
 @property (readonly, nonatomic, nonnull) NSMutableSet<ZMMessage *> *mutableMessages;
 @property (readonly, nonatomic, nonnull) NSSet<ZMMessage *> *hiddenMessages;
-@property (nonatomic, nullable) ZMConnection *connection;
```

Comment:
Not sure if I understand correctly, but can `ZMConnection` be fully removed after this migration?